crdppf/util/pdf_classes.py

```diff
@@ -296,8 +300,45 @@ def set_wms_config(self, topicid):
         else:
             self.wms_url = self.crdppf_wms
 
+    # gets the basemap to display the selected property  in the cadastral context
+    def get_basemap(self):
+        # defines the corrners and the center point of the bbox for the wms call of the map
+        wmsBBOX, wmsbbox = self.get_wms_bbox()
+
+        params = {
+            'REQUEST': 'GetMap',
+            'VERSION': self.appconfig.wms_version,
+            'LAYERS': ",".join(self.appconfig.crdppf_wms_layers),
+            'SRS': self.appconfig.wms_srs,
+            'BBOX': ",".join([str(wmsBBOX['minX']), str(wmsBBOX['minY']), str(wmsBBOX['maxX']), str(wmsBBOX['maxY'])]),
+            'WIDTH': self.mapconfig['width'],
+            'HEIGHT': self.mapconfig['height'],
+            'FORMAT': 'image/png',
+            'TRANSPARENT': 'false'
+        }
+
+        getmapurl = self.crdppf_wms
+
+        if getmapurl.find('?') < 0:
+            getmapurl += '?'
+        getmapurl = getmapurl + '&'.join(['%s=%s' % (key, value) for (key, value) in params.items()])
+
+        http = httplib2.Http()
+
+        h = dict(self.request.headers)
+        if urlparse(getmapurl).hostname != 'localhost': # pragma: no cover
```

Comment:
You should remove all the `# pragma: no cover`... This is only used if you do Travis checks on your code (what we don't do...)